src/EnergyPlus/PlantUtilities.cc

```diff
@@ -578,6 +578,13 @@ namespace PlantUtilities {
 				NodeNum = comp.NodeNumIn;
 				Node( NodeNum ).MassFlowRate = a_node_MasFlowRate;
 				Node( NodeNum ).MassFlowRateRequest = a_node_MasFlowRateRequest;
+				if ( comp.EMSLoadOverrideOn ) {
+					if ( comp.EMSLoadOverrideValue == 0.0 ) {
+						Node( NodeNum ).MassFlowRate = 0.0;
+						Node( NodeNum ).MassFlowRateRequest = 0.0;
+					}
+				}
+
```

Comment:
This does look suspect to me.  It will be okay if there is only one device on the branch (which is usually the case for coils). This routine is intended to make sure all the components on a single branch get the same flow and request as the actuated node.  This code change could mess that up with one component getting different rates than others on the same branch.  Would have to scan all the components on the branch and see if any one is shut down, and then set the same values on everything on the branch.